cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -284,6 +311,51 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
         }
     }
 
+    /**
+     * Determine if a package should be included in the download. A package may not be downloaded it the license of the
```

Comment:
Second sentence: "may" is a bit soft, also s/it/if/, and we should prefer positive logic in the formulation. How about:

A package is only downloaded if its license is part of a category that is part of the [DownloaderConfiguration]'s `includedLicenseCategories`.